src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import { User } from '../../types/types';
+import './UserInfo.scss';
+
+export type UserInfoProps = {
+  user: User;
```

Comment:
Define the user property as nullable, as it can be `null` or `undefined`. You can use the optional chaining operator `?` to achieve this: `user?: User`.